v2/pkg/subscraping/sources/fullhunt/fullhunt.go

```diff
@@ -0,0 +1,59 @@
+package fullhunt
+
+import (
+	"context"
+	"fmt"
+
+	jsoniter "github.com/json-iterator/go"
+	"github.com/projectdiscovery/subfinder/v2/pkg/subscraping"
+)
+
+//fullHunt response
+type fullHuntResponse struct {
+	Hosts   []hostDetails `json:"hosts"`
+	Message string        `json:"message"`
+	Status  int           `json:"status"`
+}
+
+// hostDetails struct
+type hostDetails struct {
+	Host string `json:"host"`
+}
+
+// Source is the passive scraping agent
+type Source struct{}
+
+func (s *Source) Run(ctx context.Context, domain string, session *subscraping.Session) <-chan subscraping.Result {
+	results := make(chan subscraping.Result)
+
+	go func() {
+		defer close(results)
+		
+		resp, err := session.Get(ctx, fmt.Sprintf("https://fullhunt.io/api/v1/domain/%s/details", domain), "", map[string]string{"X-API-KEY": session.Keys.FullHunt})
```

Comment:
Instead of the details endpoint, we should use [subdomains](https://api-docs.fullhunt.io/#get-subdomains-of-a-domain) api.

Reply:
Done.